xla/runlit.cfg.py

```diff
@@ -57,6 +57,19 @@
 else:
   llvm_config.use_default_substitutions()
 
+subst_marker = "SUBST_";
```

Comment:
I see some failures in a Kokoro job for testing OSS GPU, it looks like this config is not actually being applied. Is this actually working on your end?
Internally I got it working by adjusting our config we use internally to do the replacements. But it seems it doesn't work for open source.

Reply:
Nevermind, I think I found the problem, tensorflow/compiler/mlir/runlit.cfg.py also needs to be adjusted, it is used for tensorflow testing.

Reply:
@akuegel Thanks! Do I need to send a separate PR to the tensoflow repo?

Reply:
I am taking care of it.